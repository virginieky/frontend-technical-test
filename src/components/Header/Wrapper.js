import styled from 'styled-components';

const Wrapper = styled.div` 
  position: fixed;
  top: 0;
  width: 100%;
  z-index: 3;
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 70px;
  background-color: white;
  border-bottom: 1px solid #e9e9e9;
  div {
    position: relative;
    width: 100%;
    height: 100%;
  }
`;

export default Wrapper;
