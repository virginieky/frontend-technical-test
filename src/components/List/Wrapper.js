import styled from 'styled-components';

const Wrapper = styled.ul`
  height: calc(100vh - 130px);
  overflow: auto;
  padding-left: 0;
  margin-bottom: 0;
  list-style-type: none;
`;

export default Wrapper;
