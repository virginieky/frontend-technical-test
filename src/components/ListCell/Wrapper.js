import styled from 'styled-components';

const Wrapper = styled.li`
  width: 100%;
  height: 100px;
  flex: 0 0 auto;
  margin-bottom: 16px;
  margin-right: 16px;
`;

export default Wrapper;
