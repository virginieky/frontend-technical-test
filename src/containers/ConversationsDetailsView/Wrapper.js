import styled from 'styled-components';

const Wrapper = styled.div`
  > div {
    margin: 0 auto;
    height: calc(100vh - 70px);
    max-width: 100%;
    > div:last-of-type input {
      height: 70px;
    }
  }
`;

export default Wrapper;
