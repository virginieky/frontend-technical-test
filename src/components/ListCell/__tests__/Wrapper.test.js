import React from 'react';
import { render } from '@testing-library/react';
import Wrapper from '../Wrapper';

describe('components | ListCell | Wrapper', () => {
  it('should match the snapshot', () => {
    const { asFragment } = render(<Wrapper />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('should match the snapshot with isSelected props', () => {
    const { asFragment } = render(<Wrapper isSelected />);

    expect(asFragment()).toMatchSnapshot();
  });
});
