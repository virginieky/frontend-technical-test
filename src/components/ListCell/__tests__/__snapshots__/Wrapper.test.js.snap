// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | ListCell | Wrapper should match the snapshot 1`] = `
<DocumentFragment>
  <li
    class="Wrapper-sc-m0e7wi-0 jruAWi"
  />
</DocumentFragment>
`;

exports[`components | ListCell | Wrapper should match the snapshot with isSelected props 1`] = `
<DocumentFragment>
  <li
    class="Wrapper-sc-m0e7wi-0 jzDbvR"
  />
</DocumentFragment>
`;
